### ISSUE TITLE
Unable to sign in to Zed due to invalid certificate

### ISSUE THREAD

**Issue by galtalmor** [zed-industries/zed#7754]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When I try to sign in to Zed by clicking the "Sign in" button at the top right corner of the editor, log in to my Github account, then I get the following error in the snack bar:

```
Error: IO error: invalid certificate: UnknownIssuer

Caused by:
   0: IO error: invalid certificate: UnknownIssuer
   1: invalid certificate: UnknownIssuer
```

This is likely due to my work laptop using a Paloalto gateway with a self-signed certificate, though I was unable to find anywhere in the [settings docs](https://zed.dev/docs/configuring-zed) where I can add a certificate or skip the certificate check.

### Environment

Zed: v0.121.7 (Zed)
OS: macOS 14.3.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by JosephTLyons** [comment]
Hey @galtalmor, can you check and see if one of the newer releases still has the same issue?

https://zed.dev/releases/stable

**Comment by galtalmor** [comment]
It is resolved in the latest version (0.122.2).

Thank you!

**Comment by JosephTLyons** [comment]
> It is resolved in the latest version (0.122.2).
> 
> Thank you! 

Awesome! 🚀
